src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -126,6 +138,27 @@ private Tenant saveTenant(UUID consortiumId, Tenant tenantDto) {
     return converter.convert(savedTenant, Tenant.class);
   }
 
+  @Override
+  public UserTenant postUserTenant(UserTenant userTenant) {
+    log.info("Creating userTenant with dummy user with id {}.", userTenant.getId());
+    userTenantsClient.postUserTenant(userTenant);
+    return userTenant;
+  }
+
+  private void saveDummyUser(String tenantId) {
```

Comment:
please move java doc to this method
```suggestion
 /*
      Dummy user will be used to support cross-tenant requests checking in mod-authtoken,
      if user-tenant table contains some record in institutional tenant - it means mod-consortia enabled for
      this tenant and will allow cross-tenant request.
      
      @param tenantId tenant id
      */
  private void saveDummyUser(String tenantId) {
```

Reply:
fixed